### ISSUE TITLE
Admin screen heading tags with WordPress 4.4

### ISSUE THREAD

**Issue by lutrov** [chuckreynolds/WPFBOGP#65]
The software uses incorrect headings in admin screens. See https://make.wordpress.org/core/2015/10/28/headings-hierarchy-changes-in-the-admin-screens/ for more.

**Comment by chuckreynolds** [comment]
Yeah I've been aware of that - but not sure I even updated that in the `develop` branch.... Will get that taken care of prior to next release.
